server/routes/user.py

```
@@ -0,0 +1,83 @@
+    if not any(ch.isupper() for ch in password):
+        return False
+    if not any(ch.isdigit() for ch in password):
+        return False
+    return True
+
+
+@user_bp.route('/register', methods=['POST'])
+def register_user():
+    try:
+        payload = request.get_json(force=True, silent=False)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)